### DISCUSSION TITLE
Clicking on icon in task bar... does not minimize the app

### DISCUSSION THREAD

**Discussion by ePaulo** [jendrikseipp/rednotebook#750] [Ideas]
Currently in Windows 10, clicking on the RedNotebook icon in the task bar (when it's "maximized") does not minimize the app... as happens with other apps like VSCode.  Wondering if there is something we can add to the config or settings file(s) that will allow minimize-to-taskbar to happen.  And if the feature could be added to a future version.  Or if it's just my setup that is somehow blocking it from happening. ;-)

**Comment by jendrikseipp** [comment]
Sounds like a useful feature to have. I don't have any Windows installation at the moment, so I can't implement this myself, but I'm happy to review pull requests.
